symforce/slam/imu_preintegration/preintegrated_imu_measurements.h

```diff
@@ -0,0 +1,96 @@
+/* ----------------------------------------------------------------------------
+ * SymForce - Copyright 2022, Skydio, Inc.
+ * This source code is under the Apache 2.0 license found in the LICENSE file.
+ * ---------------------------------------------------------------------------- */
+
+#pragma once
+
+#include <Eigen/Dense>
+
+#include <sym/rot3.h>
+#include <sym/util/epsilon.h>
+
+namespace sym {
+
+/**
+ * Struct of Preintegrated IMU Measurements (not include the covariance of change in orientation,
+ * velocity, and position).
+ */
+template <typename Scalar>
+struct PIMNoCovariance {
+  using Vector3 = Eigen::Matrix<Scalar, 3, 1>;
+  using Matrix3 = Eigen::Matrix<Scalar, 3, 3>;
```

Comment:
Just use `sym::Vector3<Scalar>` and `sym::Matrix33<Scalar>`?  If we don't want to type `<Scalar>` multiple times this should at least be `Matrix33` for consistency

Reply:
Oh. Didn't actually know we had a `sym::Vector3<Scalar>` (as a side note, looks like `Eigen::Vector3<Scalar>` is added in Eigen 3.4.0).

I do like the idea of keeping the name shorted by not including `<Scalar>` to help with readability. And I'm fine with `Matrix33` as well (though, I will note that `Eigen` does `Matrix3` instead).